be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -619,6 +619,12 @@ Status ExchangeSinkOperator::push_chunk(RuntimeState* state, const ChunkPtr& chu
     return Status::OK();
 }
 
+void ExchangeSinkOperator::update_metrics(RuntimeState* state) {
+    if (_driver_sequence == 0) {
+        _buffer->update_profile(_unique_metrics.get());
```

Comment:
how much time does it cost?

Reply:
just update some atomic value. and will be invoded every 10 seconds(by default), so relatively low.